2023/README.md

```diff
@@ -1,20 +1,276 @@
+The process we went through to arrive at our priority list for Interop 2023 is available in the [planning process](planning-process.md) document.
 
-Similar to [Interop 2022](https://wpt.fyi/interop-2022), the tests selected will be continuously run on automated testing infrastructure. The test pass rates for each browser rendering engine will be displayed on the Interop 2023 Dashboard — displaying the percentage of passing tests in each chosen area, and an overall total score. The dashboard will also display scores for group progress on particular Investigation Efforts selected for the Interop team to work on throughout the year.
+Similar to [Interop 2022](https://wpt.fyi/interop-2022), the tests we selected will be continuously run on automated testing infrastructure. The test pass rates for each browser rendering engine will be displayed on the [Interop 2023 Dashboard](https://wpt.fyi/interop-2023) — displaying the percentage of passing tests in each chosen area, and an overall total score of tests that pass in every browser. The dashboard will also display scores for group progress on particular Investigation Efforts selected for the Interop team to work on throughout the year.
 
-Interop 2023 is tightly focused on technology that is already specified in web standards. This is not a place to invent new web technology, nor solve complex problems that need web standards. That work should happen in the standards bodies.
+Interop 2023 is tightly focused on technology that is already specified in web standards.
 
 ## Dashboard
-
+Our dashboard lives at [wpt.fyi/interop-2023](https://wpt.fyi/interop-2023). It computes an interop score as the percentage of tests in WPT that pass in all three browser engines.
 
 
 ## Focus Areas
 
+The folloing features were selected for Interop 2023. Descriptions are taken from issue proposal submissions and MDN documentation.
+
+
+### Border Image in CSS
+
+Authors can specify an image to be used in place of the border-style. In this case, the border’s design is taken from the sides and corners of an image specified ([CSS `border-image`](https://github.com/web-platform-tests/interop/issues/146)).
+
+[https://developer.mozilla.org/en-US/docs/Web/CSS/border-image](https://developer.mozilla.org/en-US/docs/Web/CSS/border-image)
+
+[https://w3c.github.io/csswg-drafts/css-backgrounds/#the-border-image](https://w3c.github.io/csswg-drafts/css-backgrounds/#the-border-image)
+
+[https://wpt.fyi/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-cssborderimage](https://wpt.fyi/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-cssborderimage)
```

Comment:
Linking directly to the directory where the tests are: (this isn't possible in all cases, but usually)

```suggestion
[Tests](https://wpt.fyi/results/css/css-backgrounds?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-cssborderimage)
```